vmdb/config/vmdb.tmpl.yml

```diff
@@ -70,11 +70,15 @@ ems_refresh:
     get_shared_images: true
     get_public_images: false
     ignore_terminated_instances: true
-  scvmm:
+  foreman_configuration:
     :refresh_interval: 15.minutes
+  foreman_provisioning:
+    :refresh_interval: 1.hour
```

Comment:
I think an hour is reasonable until we get the eventing in, then it should go to 24 hours.